### PR TITLE
fix README

### PR DESCRIPTION
**Pull request by WindzCUHK**
as title

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -116,23 +116,24 @@ Example:
...
-  | scope        | The scope of the access token (Only added if role is not specified) | user,                                                                                                                                                                                                                                                                                                                                                                                                                                  |
+
+| Name | Description | Example |
+| ---- | ----------- | ------- |
+| access_token | The access token generated | eyJraWQiOiIwIiwidHlwIjoiYXQrand0IiwiYWxnIjoiUlMyNTYifQ.eyJzdWIiOiJkb21haW4udHJhdmVsLnRyYXZlbC1zaXRlIiwiaWF0IjoxNTgzNzE0NzA0LCJleHAiOjE1ODM3MTY1MDQsImlzcyI6Imh0dHBzOi8venRzLmF0aGVuei5pbyIsImF1ZCI6ImRvbWFpbi5zaG9wcGluZyIsImF1dGhfdGltZSI6MTU4MzcxNDcwNCwidmVyIjoxLCJzY3AiOlsidXNlcnMiXSwidWlkIjoiZG9tYWluLnRyYXZlbC50cmF2ZWwtc2l0ZSIsImNsaWVudF9pZCI6ImRvbWFpbi50cmF2ZWwudHJhdmVsLXNpdGUifQ.\[signature] |
+| token_type   | The token type of the access token | Bearer |
+| expires_in   | The expiry duration of the access token | 1000 |
```
Comment (kevindiu) [inline README.md]: better to specify second?

Comment (WindzCUHK) [reply README.md]: fixed: https://github.com/yahoojapan/athenz-client-sidecar/pull/39/commits/954e3400fbfcf07cc972f556e49abd6af1cbbf7e

File: README.md [hunk 2]
```diff
@@ -116,23 +116,24 @@ Example:
...
-  | scope        | The scope of the access token (Only added if role is not specified) | user,                                                                                                                                                                                                                                                                                                                                                                                                                                  |
+
+| Name | Description | Example |
+| ---- | ----------- | ------- |
+| access_token | The access token generated | eyJraWQiOiIwIiwidHlwIjoiYXQrand0IiwiYWxnIjoiUlMyNTYifQ.eyJzdWIiOiJkb21haW4udHJhdmVsLnRyYXZlbC1zaXRlIiwiaWF0IjoxNTgzNzE0NzA0LCJleHAiOjE1ODM3MTY1MDQsImlzcyI6Imh0dHBzOi8venRzLmF0aGVuei5pbyIsImF1ZCI6ImRvbWFpbi5zaG9wcGluZyIsImF1dGhfdGltZSI6MTU4MzcxNDcwNCwidmVyIjoxLCJzY3AiOlsidXNlcnMiXSwidWlkIjoiZG9tYWluLnRyYXZlbC50cmF2ZWwtc2l0ZSIsImNsaWVudF9pZCI6ImRvbWFpbi50cmF2ZWwudHJhdmVsLXNpdGUifQ.\[signature] |
+| token_type   | The token type of the access token | Bearer |
+| expires_in   | The expiry duration (in second) of the access token | 1000 |
```
Comment (kevindiu) [inline README.md]: maybe better to copy the sentence here?

https://github.com/yahoojapan/athenz-client-sidecar/pull/39/files#diff-04c6e90faac2675aa89e2176d2eec7d8R110

Comment (WindzCUHK) [reply README.md]: fixed: 6ddc184

**Pull request closed without merging**